src/platform/packages/shared/kbn-unified-tabs/src/components/tabs_bar/tabs_bar.tsx

```diff
@@ -96,6 +98,16 @@ export const TabsBar = forwardRef<TabsBarApi, TabsBarProps>(
     tabsContainerRef.current = tabsContainerElement;
     const hasReachedMaxItemsCount = maxItemsCount ? items.length >= maxItemsCount : false;
     const moveFocusToItemIdRef = useRef<string | null>(null);
+    const hasSentMaxReachedEventRef = useRef(false);
+
+    const emitOnKeyUsedEvent = useCallback(
+      (key: string) => {
+        onEvent('tabsKeyboardShortcutsUsed', {
+          keyUsed: key,
+        });
+      },
+      [onEvent]
+    );
```

Comment:
It looks like we're not capturing any of the multi-key shortcuts yet, but maybe we should prepare for it anyway by using shortcut names like `moveLeft`, `moveEnd`,`close`, etc. instead of the keys? That would also keep the telemetry consistent in case we ever change a shortcut key.

Reply:
Sounds good to me, I'll adjust!

Reply:
Addressed in [5d69a7b](https://github.com/elastic/kibana/pull/234344/commits/5d69a7bc65e9cb849ec228a5b0f8a7eb04010a47)